typescript/sdk/src/index.ts

```diff
@@ -53,6 +53,12 @@ export {
   HyperlaneContractsMap,
   HyperlaneFactories,
 } from './contracts/types';
+export { CosmWasmCoreAdapter } from './core/adapters/CosmWasmCoreAdapter';
```

Comment:
Why did the import orders in this file change?

Reply:
it seems lexicographically correct I think?